src/Compilers/CSharp/Portable/Parser/Lexer_StringLiteral.cs

```diff
@@ -487,15 +504,7 @@ private void ScanFormatSpecifier()
                     }
                     else if (ch == '}')
                     {
-                        if (_lexer.TextWindow.PeekChar(1) == '}')
```

Comment:
i actually don't think this goes far enough.  If i look at the runtime impl, they just fail on usign a `{` in a format string as well.  I think we should follow that logic ourselves in the code that appears above this on line 474.

Reply:
You can see the runtime logic here that errors: https://github.com/dotnet/runtime/blob/c72678bac90f9692ccea2787e22d88a6ae449cda/src/libraries/System.Private.CoreLib/src/System/Text/StringBuilder.cs#L1706

Reply:
this also matches teh grammar:

```
interpolation_format_character
    : '<Any character except \" (U+0022), : (U+003A), { (U+007B) and } (U+007D)>'
```

Reply:
An error is now emitted (and I add tests for those)